tests/tests.go

```diff
@@ -232,14 +236,14 @@ func DeleteLeaderManager() {
 
 func changeManagerReplicas(numOfReplica int32) error {
 	Eventually(func() error {
-		managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(ManagerNamespce).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+		managerStatefulset, err := testClient.KubeClient.AppsV1().StatefulSets(ManagerNamespce).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
```

Comment:
please change names.MANAGER_DEPLOYMENT const to names.MANAGER_STATEFULSET

Reply:
As far as I understand, this const represent the manager deployment name, and by deployment I mean the action but not the method (whether it is done with Deployment, StatfulSet or any other kind).
We can change it to MANAGER_PODS_NAME,
because eventually the name of the pods will be  derivative of this const value, or just leave it.
What you think?

Reply:
the pod's name is derived from the statefulset/deployment, so I wouldn't change it to that.

I rechecked and names.MANAGER_DEPLOYMENT is currently only used to retrieve the statefulset object. 

I think changing it might reduce future confusion, but if you're adamant then I leave the call to you.